src/tribler-core/tribler_core/modules/tests/test_versioncheck.py

```diff
@@ -97,3 +97,40 @@ async def test_non_json_response(self):
             versioncheck_manager.check_failed = True
 
         self.assertTrue(versioncheck_manager.check_failed)
+
+    @timeout(10)
+    async def test_version_check_timeout(self):
+        await self.setup_version_server(json.dumps({'name': 'v1337.0'}))
+
+        self.new_version_called = False
+        # Default timeout is 5 seconds so under normal circumstance, we don't expect timeout
+        await self.session.version_check_manager.check_new_version()
+        self.assertTrue(self.new_version_called)
```

Comment:
Thanks for the wonderful `Mock` magic, it is now much easier to check if the notifier was called as the result of version checking procedure. See 
https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.assert_called_once_with

Please, rewrite the version checker tests using `Mock` object.

Reply:
updated